front/app/utils/moduleUtils.ts

```diff
@@ -323,7 +323,10 @@ export type OutletsPropertyMap = {
     onSkip: (name: IOnboardingCampaignNames) => void;
     onAccept: (name: IOnboardingCampaignNames) => void;
   };
-  'app.containers.Admin.settings.customize.fields': {
+  'app.containers.App.signUpInModal': {
```

Comment:
Why have you added this line?
Is this outlet used somewhere? https://github.com/CitizenLabDotCo/citizenlab/search?q=app.containers.App.signUpInModal

Reply:
Was accidentally because I was trying out things for a different I think. Thanks for spotting!